luminance/Cargo.toml

```
@@ -19,6 +19,8 @@ maintenance = { status = "actively-developed" }
 default = ["std"]
 std = ["gl"]
 
+multiwin = []
```

The name of this featuree-flag is not really self-explanatory. Please switch to something more explicit.

---

I agree. 'multi' must be keep, because of the idea behind, but as you can use multiple contexts without multiple windows, I suggest this evolution:

```suggestion
multi-contexts = []
```

What do you think about this change?